pyproject.toml

```
@@ -26,7 +26,8 @@ dependencies = [
     "omegaconf",
     "opencv-python",
     "gymnasium",
-    "lanelet2"
+    "lanelet2",
+    "torchdrivesim"
```

```suggestion
    "torchdrivesim",
```
Makes for nicer diffs in the future

---

Done.